### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by TheKnightGamer** [ImagicalDevs/UltimateSurvivalGames#9]
@ImagicalDevs  So we need some help to complete this plugin.

**Comment by AmiroEmad** [comment]
Todo List:
[] multispawn system

[] countdown timer before start

[] multiworld support

[] invinsabilty 10 seconds when start

**Comment by TheKnightGamer** [comment]
Yeah let's finish this plugin then we can move on with our lifes.
